models/requests.js

```diff
@@ -0,0 +1,59 @@
+module.exports = function(sequelize, DataTypes) {
+    var Request = sequelize.define("Request", {
+      // recipient_id: {
+      //   type: DataTypes.INTEGER,
+      //   primaryKey:true,
+      //   allowNull: false,
+      //   autoIncrement: true,
+      //   unique: true
+      // },
+      // services_id: {
+      //   type: DataTypes.INTEGER,
+      //   primaryKey:true,
+      //   allowNull: false,
+      //   autoIncrement: true,
+      //   unique: true
+      // },
+      finalized: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: false
+      },
+      detail: {
+        type: DataTypes.TEXT,
+        allowNull: false,
+        validate: {
+          len: [1]
+        }          
+      },
+      requester_rating: {
+        type: DataTypes.INTEGER,
+        allowNull: true
+      },
+      bounty_points: {
+        type: DataTypes.INTEGER,
+        allowNull: true,
+      },
+      start_date: {
+          type: DataTypes.DATE,
+          allowNull: true
+      },
+      end_date: {
+          type: DataTypes.DATE,
+          allowNull: true
+      }
```

Comment:
I'm realizing we may want the following columns added for Requests:
```MySQL
address VARCHAR(255),
city VARCHAR(100),
state VARCHAR(2),
zipcode INT,
```
They should have `allowNull: true` as well.